open_spiel/python/examples/benchmark_games.py

```diff
@@ -22,6 +22,8 @@
 from absl import logging
 import pandas as pd
 
+from open_spiel.python import games  # pylint: disable=unused-import
+from open_spiel.python.mfg import games as mfg_games  # pylint: disable=unused-import
```

Comment:
You will need to handle mean field nodes for MFGs to work in the benchmark, but this can wait another pull request.

Reply:
See remark from Julien just below.